query-planner-js/src/composedSchema/toAPISchema.ts

```diff
@@ -14,27 +14,31 @@ declare module 'graphql' {
   }
 }
 
-export function toAPISchema(schema: GraphQLSchema): GraphQLSchema {
+export function toAPISchema(schema: GraphQLSchema): {
```

Comment:
big nit (can be a follow-up as it's internal ~enough): i'm questioning the naming of this function a bit.  

reason: a `toAPISchema` function which returns an object with a `schema` property is less intuitively named to me than `toAPISchema` was.  Maybe just `toAPI`?

Reply:
Just to clarify, by _internal ~enough_ I was alluding to our cloud's usage.  We're close enough to that work to easily change it without fear.  We can't unnecessarily block this from landing right now.

Reply:
outdated